lib/tasks_private/openstack.rake

```diff
@@ -1,37 +1,85 @@
-require_relative 'vcr_recorder'
-
 namespace :vcr do
-  namespace :credentials do
-    desc 'Load credentials from openstack_environments.yml'
-    task :load do
-      VcrRecorder.new.load_credentials
-    end
-    desc 'Obfuscate real credentials from specs and cassettes'
-    task :obfuscate do
-      VcrRecorder.new.obfuscate_credentials
-    end
-  end
-
   namespace :cassettes do
     desc 'Deletes VCR cassettes for OpenStack Cloud Provider'
-    task :delete do
-      VcrRecorder.new.delete_cassettes
+    task :delete => :environment do
+      vcr_dir = ManageIQ::Providers::Openstack::Engine.root.join("spec/vcr_cassettes/manageiq/providers/openstack/cloud_manager")
+      Dir.glob(vcr_dir.join("refresher*.yml")).each { |f| File.delete(f) }
     end
   end
 
   namespace :spec do
     desc 'Run specs needed for rerecording of VCRs'
-    task :run do
-      ENV['SPEC'] = VcrRecorder.new.test_files.join(' ')
+    task :run => :environment do
+      ENV['SPEC'] = Dir.glob(ManageIQ::Providers::Openstack::Engine.root.join("spec/models/manageiq/providers/openstack/cloud_manager/refresher_spec.rb")).first
       Rake::Task['spec'].invoke
     end
   end
 
   desc 'Rerecord all of VCR cassettes'
-  task :rerecord => [] do
+  task :rerecord => :environment do
+    require 'fog/openstack'
+
+    host, port, username, password = YAML.load_file("config/secrets.yml").dig("test", "openstack").values_at("hostname", "port", "userid", "password")
+
     Rake::Task['vcr:cassettes:delete'].invoke
-    Rake::Task['vcr:credentials:load'].invoke
-    Rake::Task['vcr:spec:run'].invoke
-    Rake::Task['vcr:credentials:obfuscate'].invoke
+
+    fog_connect_opts = {
+      :provider               => "OpenStack",
+      :openstack_auth_url     => "https://#{host}:#{port}/v3/",
+      :openstack_username     => username,
+      :openstack_api_key      => password,
+      :openstack_project_name => "admin",
+      :openstack_domain_id    => "default",
+      :connection_options     => {:ssl_verify_peer => false}
+    }
+
+    compute = Fog::Compute.new(fog_connect_opts)
+    network = Fog::Network.new(fog_connect_opts)
+
+    begin
+      puts "Creating resources..."
+      cn = network.create_network(:name => "test-network")
```

Comment:
If you do
```suggestion
      cn = network.create_network(:name => "test-network").body["network"]
```
then `cn` will be the actual cloud network body not the entire response object